### PR TITLE
Addition of Icons

### PR DESCRIPTION
**Pull request by hellcp**
To have them visible and recognizable, icons have been divided into several resolutions, even though all of them are SVG, there is significant difference between visibility of shadows, highlights and contours in different resolutions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Fest ![Logo](/res/icons/hicolor/24x24/apps/fest.svg)
+# Fest <img src="/res/icons/hicolor/24x24/apps/fest.svg" height="24" width="24">
```
Comment (jcgruenhage) [inline README.md]: What's the point of this change? Having this as MD is preferable to having it as HTML.

**Review by jcgruenhage — CHANGES_REQUESTED** [review]
This is great, but I disagree with having HTML in the README file. @jplatte @her001 any thoughts on that?

File: README.md (hunk 1 above)
Comment (hellcp) [reply README.md]: Compat with browsers, they don't always respect size of SVG and require size provided to them

**Review by jcgruenhage — APPROVED** [review]

**Comment by jplatte** [comment]
🎉

**Pull request merged**